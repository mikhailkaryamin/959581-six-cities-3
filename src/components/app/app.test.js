import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";
import configureStore from "redux-mock-store";
import {
  Provider
} from "react-redux";
import {
  storeWithoutActiveOffer
} from "../../mocks/initialState.js";

const mockStore = configureStore([]);

describe(`App render correctly`, () => {
  it(`App render correctly with active offer`, () => {
    const store = mockStore(storeWithoutActiveOffer);
    const tree = renderer
      .create(
          <Provider
            store={
              store
            }
          >
            <App />
          </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
