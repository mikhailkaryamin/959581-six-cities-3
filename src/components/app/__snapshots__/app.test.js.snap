// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render correctly App render correctly with active offer 1`] = `null`;
